room/compiler-processing/src/main/java/androidx/room/compiler/processing/XProcessingEnv.kt

```diff
@@ -141,12 +160,14 @@ interface XProcessingEnv {
             options: Map<String, String>,
             resolver: Resolver,
             codeGenerator: CodeGenerator,
-            logger: KSPLogger
+            logger: KSPLogger,
+            targetLanguage: Language = Language.KOTLIN
```

Comment:
we should have the same default for all and since it works like Java now, this should always be JAVA.

Reply:
Yeah, that´s true, changed that part.